.php_cs.dist

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$finder = new PhpCsFixer\Finder();
+$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
+$finder->in(__DIR__);
+
+/* Based on ^2.1 of php-cs-fixer */
+$config
+    ->setRules([
+        // default
+        '@PSR2' => true,
```

Comment:
Unfortunately php-cs-fixer does not play nice together with HHVM. Since HHVM is one of the required versions in the matrix that must pass, I have to resort to dynamically installing php-cs-fixer. I opted to only install it on one specific build (chose the same one that is used for codecoverage, but can also move it to 7.1 if preferred).